src/components/PendingTxsSnackbar/store.test.js

```diff
@@ -0,0 +1,317 @@
+import { TransactionType, TransactionStatus } from 'constants';
+import { observable } from 'mobx';
+import PendingTxsSnackbarStore, { INIT_VALUES } from './store';
+import { mockResetTransactionList, mockAddTransaction } from '../../network/graphql/queries/';
+
+jest.mock('../../network/graphql/queries'); // block and manually mock our backend
+
+describe('PendingTxsSnackbarStore', () => {
+  let store;
+  const app = {
+    global: observable({ syncBlockNum: 0, syncPercent: 10 }),
+  }; // mock the appstore
+  const sleep = ms => new Promise(res => setTimeout(res, ms));
+
+  describe('Init', () => {
+    beforeEach(() => {
```

Comment:
You can move this entire `beforeEach()` to the root level `describe`. This will run for each test then.

Reply:
also, shouldn't every test have `init()` called?